map_list:
  - map:
    id: 0
    pose_x:
    pose_y:
    orientation_z:
    orientation_w: